packages/internal/src/project.ts

```diff
@@ -0,0 +1,42 @@
+import fs from 'fs'
+import path from 'path'
+
+import { getPaths } from './paths'
+
+// TODO: make this dynamic
+export enum SidesEnum {
+  API = 'api',
+  WEB = 'web',
+}
+
+/**
+ * Returns array of a project's existing sides
+ */
+export const getSides = (): string[] => {
```

Comment:
Not a big fan of this lint rule that forces you declare return types. Inferred types are better for long term maintenance because you don't have to change them as function definition changes. 
```suggestion
export const getSides = () => {
```